### PR TITLE
Page Navigation Enhancements 

### LINKED ISSUE #42: Site Navigation Improvements
**Is your feature request related to a problem? Please describe.**
We can only move one page at a time. No nav link on the logo. When the user clicks next or previous page it keeps the page scrolled down. 

**Describe the solution you'd like**
Add first and last page buttons, an anchor tag on the log, and a scroll to the top feature on the pagination buttons.

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. First and last page buttons
2. Link on the logo that takes user to home page
3.  Scroll to the top of the page when switching pages

## Purpose
Improve the user experience when navigating between pages

## Approach
Rendered first and last page buttons. Put an anchor tag linked to the home page on the logo. Made a new function to scroll users to the top of the page when they switch pages. 

Closes #42

### CODE REVIEW

File: movie-app/src/body/Searched.js [hunk 1]
```diff
@@ -1,52 +1,104 @@
...
+  movies,
+  setPage,
+  page,
+  toggleModal,
+  selected,
+  setSelected,
+  modalOpen,
+  setModalOpen,
+  detailClick,
+}) => {
```
Comment (kmausolf) [inline movie-app/src/body/Searched.js:15]: Good readability refactor!

File: movie-app/src/body/Searched.js [hunk 2]
```diff
@@ -1,52 +1,104 @@
...
-        } else {
-            return <div>Loading...</div>;
-        }
-    };
+  const scrollToTop = () => {
+    window.scrollTo({
+      top: 0,
+      behavior: "auto",
+    });
+  };
```
Comment (kmausolf) [inline movie-app/src/body/Searched.js:39]: Very nice, it's helpful to annotate common behavior like this with a function.

File: movie-app/src/body/Searched.js [hunk 3]
```diff
@@ -1,52 +1,104 @@
...
 
-    let movieRemainder = parseInt(totalResults) % 10 === 0 ? parseInt(totalResults) / 10 : Math.floor(parseInt(totalResults) / 10 + 1)
+  let movieRemainder =
+    parseInt(totalResults) % 10 === 0
+      ? parseInt(totalResults) / 10
+      : Math.floor(parseInt(totalResults) / 10 + 1);
```
Comment (kmausolf) [inline movie-app/src/body/Searched.js:44]: Another nice readability refactor.  It's generally a good idea to not have lines longer than 80 characters.

File: movie-app/src/header/NavBar.js [hunk 4]
```diff
@@ -27,87 +27,90 @@ const NavBar = ({ setPage, movieSearch, setMovieSearch }) => {
...
+            onClick={(e) => {
+              closeSwitch(e);
+              clearText();
+              setMovieSearch();
+            }}
+            className={applyStyles(`cancel`)}
+            src={cancel}
+            alt="cancel-icon"
+          />
+        </div>
```
Comment (Lap343) [inline movie-app/src/header/NavBar.js:104]: Pretty makes the code look way better

**Review by Lap343 — APPROVED** [review]
Looks good

**Review by Gjhnsn — APPROVED** [review]
Good work!

File: movie-app/src/header/NavBar.js [hunk 5]
```diff
@@ -27,87 +27,90 @@ const NavBar = ({ setPage, movieSearch, setMovieSearch }) => {
...
+  };
 
   return (
     <div className="header">
       <nav className="navigation">
         <div className="logo-container">
-          <img src={"logo.png"} alt="ToadTv logo" className="logo"  />
+          <a href="/">
+            <img src={"logo.png"} alt="ToadTv logo" className="logo" />
+          </a>
```
Comment (kmausolf) [inline movie-app/src/header/NavBar.js:72]: Nice!  Users expect that clicking the site logo will return them to the homepage.

**Review by gonzalespaulb — APPROVED** [review]
LGTM

**Review by kmausolf — APPROVED** [review]
Great work on this, and thanks for using a code formatter!  It's nice to have consistency across the files.

**Pull request merged**